code_challenges/hacker_rank/nested_lists.py

```diff
@@ -0,0 +1,32 @@
+def student_grades(records):
+
+    lowest_score = 100
+    second_lowest_score = 100
+    lowest_students = []
+    second_lowest_students = []
+    for student_name_and_score in records:
+        if student_name_and_score[1] < lowest_score:
+            second_lowest_score = lowest_score
+            lowest_score = student_name_and_score[1]
+            second_lowest_students = lowest_students
+            lowest_students = [student_name_and_score[0]]
+        elif student_name_and_score[1] == lowest_score:
+            lowest_students.append(student_name_and_score[0])
+        elif student_name_and_score[1] < second_lowest_score:
+            second_lowest_students = [student_name_and_score[0]]
+            second_lowest_score = student_name_and_score[1]
+        elif student_name_and_score[1] == second_lowest_score:
+            second_lowest_students.append(student_name_and_score[0])
+    if second_lowest_students == []:
+        return "No Second Lowest"
+    else:
+        return "\n".join([name for name in sorted((second_lowest_students))])
```

Comment:
What all in `return "\n".join([name for name in sorted((second_lowest_students))])
Write` is needed?

Did you try `"\n".join(sorted(second_lowest_students))`?